pkg/agent/datamodel/sig_config.go

```diff
@@ -447,6 +452,20 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGUbuntuMinimalContainerd2204ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuResourceGroup,
+		Gallery:       AKSUbuntuGalleryName,
+		Definition:    "2204minimalcontainerd",
+		Version:       "202306.30.0",
```

Comment:
can we have a separate const for minimal versions?

Reply:
I had thought so but its gonna be testing only after which the version should be same as linuxSIGimage version, hence I did not bother creating a different json file.

Reply:
makes sense, though I was mainly just talking about a const in the go file here, no need to deal with the json embedding - either way is fine tho

Reply:
will let it stay as is if it isnt blocking. will ofcourse revisit once its actively being used/built.